features/sketch/src/main/java/com/wire/android/feature/sketch/DrawingCanvasBottomSheet.kt

```diff
@@ -17,70 +17,209 @@
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
 import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.wire.android.feature.sketch.model.DrawingState
+import com.wire.android.model.ClickBlockParams
+import com.wire.android.ui.common.Icon
+import com.wire.android.ui.common.button.IconAlignment
+import com.wire.android.ui.common.button.WireButtonState
+import com.wire.android.ui.common.button.WirePrimaryIconButton
+import com.wire.android.ui.common.button.WireSecondaryButton
+import com.wire.android.ui.common.button.WireSecondaryIconButton
+import com.wire.android.ui.common.button.WireTertiaryIconButton
+import com.wire.android.ui.common.button.wireSendPrimaryButtonColors
+import com.wire.android.ui.common.colorsScheme
+import com.wire.android.ui.common.dimensions
+import com.wire.android.ui.theme.wireDimensions
+import com.wire.android.ui.theme.wireTypography
+import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.launch
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun DrawingCanvasBottomSheet(
     onDismissSketch: () -> Unit,
-    onSendSketch: () -> Unit
+    onSendSketch: (Uri) -> Unit,
+    tempWritableImageUri: Uri?,
+    conversationTitle: String = "",
+    viewModel: DrawingCanvasViewModel = viewModel(),
 ) {
-    val sheetState = rememberModalBottomSheetState(skipPartiallyExpanded = true)
+    val context = LocalContext.current
+    val scope = rememberCoroutineScope()
+    val sheetState = rememberModalBottomSheetState(skipPartiallyExpanded = true, confirmValueChange = { false })
     ModalBottomSheet(
+        shape = CutCornerShape(dimensions().spacing0x),
+        containerColor = colorsScheme().background,
         dragHandle = {
+            DrawingTopBar(scope, sheetState, conversationTitle, onDismissSketch, viewModel::onUndoLastStroke, viewModel.state)
+        },
+        sheetState = sheetState,
+        onDismissRequest = onDismissSketch
+    ) {
+        Column(modifier = Modifier.fillMaxSize()) {
+            Row(Modifier.weight(weight = 1f, fill = true)) {
+                DrawingCanvasComponent(
+                    state = viewModel.state,
+                    onStartDrawingEvent = viewModel::onStartDrawingEvent,
+                    onDrawEvent = viewModel::onDrawEvent,
+                    onStopDrawingEvent = viewModel::onStopDrawingEvent,
+                    onSizeChanged = viewModel::onSizeChanged,
+                    onStartDrawing = viewModel::onStartDrawing,
+                    onDraw = viewModel::onDraw,
+                    onStopDrawing = viewModel::onStopDrawing
+                )
+            }
             Row(
```

Comment:
same here

Reply:
Good catch, all of these, were leftovers from an old approach I was trying to get the bottom sheet not draggable.